styleguide/derek/global-components/filter-bar/filter-accordion.js

```diff
@@ -0,0 +1,64 @@
+// Filter bar plugin
+(($) => {
+  $.fn.rsFilterBarAccordion = function rsFilterBarAccordion() {
+    const $filterBar = $(this);
+    const $window = $(window);
+    const activeClass = 'rsFilter-sideBar-active';
+    const contentClass = '.rsFilter-sideBar-accItemContent';
+    const labelClass = '.rsFilter-sideBar-accLabel';
+    const $item = $filterBar.find(labelClass);
+    const $hamburger = $filterBar.find('.rsFilter-sideBar-hamburger');
+    const $formContent = $filterBar.find('.rsFilter-sideBar-content');
+    const frameRate = 150;
+
+    const actions = {
+      openActive($el) {
+        // close all siblings
+        $.when(this.closeAll($el)).done(() => {
+          // then open content for clicked item
+          const $content = $el.next(contentClass);
+          $content.slideToggle(frameRate, () => {
+            if ($content.is(':visible')) {
+              $el.addClass(activeClass);
+            } else {
+              $el.removeClass(activeClass);
+            }
+          });
+        });
+      },
+      closeAll($el) {
+        // some promise here
+        const $siblings = $el.parent().siblings();
+        $siblings.find(labelClass).removeClass(activeClass);
+        $siblings.find(contentClass).slideUp(frameRate);
+      },
+      toggleForm() {
+        $formContent.slideToggle(frameRate);
+      },
+      resizeTimer: null,
+      resetForm() {
+        clearTimeout(this.resizeTimer);
+        this.resizeTimer = setTimeout(() => {
+          if ($window.outerWidth() > 768 && !$formContent.is(':visible')) {
```

Comment:
Should this be `>= 768`?  We change viewports at exactly 768px so it seems we want to be inclusive of that.